GEOSsolar_GridComp/GEOS_SolarGridComp.F90

```diff
@@ -377,6 +379,14 @@ subroutine SetServices ( GC, RC )
       NUM_BANDS_SOLAR = NB_CHOU
     end if
 
+    ! Decide if should make OBIO exports
+    if (USE_RRTMG) then
+       call ESMF_ConfigGetAttribute(CF, RRTMG_TO_OBIO, LABEL='RRTMG_TO_OBIO:', &
```

Comment:
Ugh. I might make a future PR to convert these to `MAPL_GetResource`. But for now, get in the tested code!

Reply:
Another one of my branches fixes these and converts them to MAPL_GetResource. But that branch is in limbo right now.

Reply:
I was in a hurry and just copying the existing usage of ESMF_ConfigGetAttribute in the Solar Set services.

Reply:
Okay! Good enough for me :)